### DISCUSSION TITLE
QR Code scanning via webcam or phone cam

### DISCUSSION THREAD

**Discussion by spierepf** [Budibase/budibase#5012] [Ideas]
I'm looking at implementing an app in budibase that uses QR codes to identify physical assets. Is it possible for a budibase app to use a phone's camera or a desktop webcam to scan QR codes?

**Comment by mjashanks** [comment]
Hey @spierepf - it's not possible at the moment, but something that we'd very much like to add. Just logged as feature request: https://github.com/Budibase/budibase/issues/5020
